book/fundamentals/README.md

```diff
@@ -0,0 +1 @@
+# fundamentals
```

Comment:
May not need this as a separate page, could consider removing in favor of `book/fundamentals/introduction`

Reply:
OK